syncs/github-issues/main.ts

```diff
@@ -46,7 +46,7 @@ const octokit = new OctokitWithGrapQLPagination({
 const issuesBuffer = [];
 
 const iterator = octokit.graphql.paginate.iterator(query, {
-    owner, repo, perPage: 30
+    owner, repo, perPage: 100
```

Comment:
@caugner could we make this configurable thru the `MERGESTAT_PARAMS` env as well?

Reply:
Fixed in 2429c3c.